sprint-guide/resources/index.njk

```diff
@@ -4,9 +4,11 @@ permalink: "sprint-guide/resources/index.html"
 eleventyExcludeFromCollections: true
 ---
 
-<h1>Resources</h1>
-<article class="measure">
-  {% renderFile './sprint-guide/resources/_introduction.md' %}
-</article>
-
-{% renderFile './sprint-guide/resources/_list.md' %}
+<div class="internal-page">
```

Comment:
I'm wondering if we can move applying this wrapper/class to a layout? That way we add `layout: layouts/internal-page` to any pages we want to apply it to.